Hymns/AppDelegate.swift

```diff
@@ -22,6 +23,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Called when the user discards a scene session.
         // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
         // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+
+        //Handle Realm Migrations https://realm.io/docs/swift/latest/#migrations
+        let config = Realm.Configuration(
+            // Set the new schema version. This must be greater than the previously used
+            // version (if you've never set a schema version before, the version is 0).
+            schemaVersion: 1,
+
+            // Set the block which will be called automatically when opening a Realm with
+            // a schema version lower than the one set above
+            migrationBlock: { migration, oldSchemaVersion in
+                // We haven’t migrated anything yet, so oldSchemaVersion == 0
+                if oldSchemaVersion < 1 {
+                    migration.enumerateObjects(ofType: TagEntity.className()) { _, newObject in
+                        newObject!["privateTagColor"] = 0
+                    }
+                    // Nothing to do!
+                    // Realm will automatically detect new properties and removed properties
```

Comment:
I'm a bit confused with the statement `Realm will automatically detect new properties and removed properties`. What does that mean? Does that mean we don't need to initialize the new properties?

Reply:
Sorry, lol thats not an accurate comment. I went ahead and deleted it. We definitely do need to initialize new properties.